server/actions/reactivateUser.js

```diff
@@ -3,8 +3,7 @@ import {connect} from 'src/db'
 const r = connect()
 
 export default async function reactivateUser(id) {
-  return r.table('users')
-    .get(id)
-    .update({active: true, updatedAt: r.now()}, {returnChanges: true})
-    .run()
+  const getUser = r.table('users').get(id)
+  await getUser.update({active: true, updatedAt: r.now()}).run()
+  return await getUser.run()
```

Comment:
Note that it's not necessary to have the `await` if you're just returning the value of the resolved promise. async functions always return promises, so:
```js
return await getUser.run()
```
is effectively the same as just having:

```js
return getUser.run()
```